tests/Backend/Mixed/SharingTest.php

```diff
@@ -1200,6 +1231,274 @@ function (array $column) {
         $this->assertCount(5, $workspaceTableData);
     }
 
+    private function sharedBucketDetailTest($bucketId, $tableId, $project)
+    {
+        $columnId = $tableId . '.transid';
+        $this->prepareTestMetadata($bucketId, $tableId, $columnId);
+
+        $response = $this->_client2->getSharedBucketDetail(
```

Comment:
co přináší tenhle test, v tom druhém taháš všechno. Je potřeba to dělat 2x?

Reply:
Jo ja som to vlastne myslel tak, ze chcem otestovat ze ak tam nedam `include=metadata` a pod. ze to nebude v response, ale jak na to teraz pozeram ten `'include' => 'columns'`tam je naviac tak ho dostranim a budem testovat ci tam je tables (pretoze ten tam je vzdy) a ze tam niesu prave tie ostatne veci

Reply:
Takto to pak dostava zmysel, ze si otestujem ze tam nieje nic ine len tables, a pak testujem vsetko ostatne.

Reply:
Jo, jasně. Dává smysl. OK.